core/templates/components/state-editor/state-responses-editor/state-responses.component.html

```diff
@@ -32,7 +32,7 @@ <h3 class="oppia-exp-answer-card-header">Learner's Answers and Oppia's Responses
 
               <div class="oppia-rule-header-warning-placement" *ngIf="isSelfLoopThatIsMarkedCorrect(answerGroup.outcome) || isSelfLoopWithNoFeedback(answerGroup.outcome)" (click)="changeActiveAnswerGroupIndex(index)"
                    ngbTooltip="{{getOutcomeTooltip(answerGroup.outcome)}}"
-                   placement="bottom">
+                   placement="right">
```

Comment:
Should we rather use `"auto"` to be safe that this is never cropped by accident? https://ng-bootstrap.github.io/#/guides/positioning#api

Reply:
ditto elsewhere

Reply:
done